ExperimentTwitter/TweetsTableViewController.m

```diff
@@ -0,0 +1,174 @@
+//
+//  TweetsViewController.m
+//  ExperimentTwitter
+//
+//  Created by ayur.j on 25/08/17.
+//  Copyright © 2017 ayur.j. All rights reserved.
+//
+
+#import "TweetsTableViewController.h"
+#import "PostTweetViewController.h"
+#import "CustomTweetCell.h"
+#import "TwitterClient.h"
+#import "Tweet+Twitter.h"
+#import "TweetsSearchViewController.h"
+
+@interface TweetsTableViewController ()
+@property (strong, nonatomic) NSMutableArray *tweets;
```

Comment:
This property should be readonly.

Reply:
Need to set and modify tweets array at different places.